resources/TrialApp.didact.md

```diff
@@ -1,4 +1,4 @@
-# Welcome to your Trial App Didact Tutorial
+# Welcome to your Trial App
```

Comment:
Not Trial App Experience?

Reply:
Matt requested this, I don't know if either of us considered putting "Trial App Experience"

Reply:
Hahaha, I'm just teasing a little since it was in your presentation